stix2elevator/missing_policy.py

```diff
@@ -204,13 +232,29 @@ def statement_type_as_extension_properties(container, statement, property_name,
         add_confidence_property_as_extension_property(map, statement.confidence, property_name, id)
     if map:
         if statement.value:
-            map["value"] = str(statement.value)
+            value_as_string = str(statement.value)
+            if is_literal:
+                if value_as_string in mapping:
+                    map["value"] = mapping[value_as_string]
+                else:
+                    map["value"] = convert_to_stix_literal(value_as_string)
+            else:
+                map["value"] = value_as_string
         container[property_name] = [map] if is_list else map
     else:
-        container[property_name] = [str(statement.value)] if is_list else str(statement.value)
+        if statement.value:
+            value_as_string = str(statement.value)
+            if is_literal:
+                if value_as_string in mapping:
+                    converted_value = mapping[value_as_string]
+                else:
+                    converted_value = convert_to_stix_literal(value_as_string)
+                container[property_name] = [converted_value] if is_list else converted_value
+            else:
+                container[property_name] = [value_as_string] if is_list else value_as_string
 
 
-def handle_missing_statement_properties(container, statement, property_name, id, is_list=False):
+def handle_missing_statement_properties(container, statement, property_name, id, is_list=False, is_literal=True, mapping={}):
```

Comment:
Not a good idea to use an empty list or dictionary as a default parameter: https://newbedev.com/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Better is to use `mapping=None` and do
```python
if mapping is None:
    mapping = {}
```
in the function body.

Reply:
Done...

I wonder if this is overkill when the mapping will not be altered within the method...